pynamodb/tests/test_model.py

```diff
@@ -2413,3 +2423,104 @@ def test_loads(self):
             ]
         }
         self.assert_dict_lists_equal(req.call_args[0][1]['RequestItems']['UserModel'], args['UserModel'])
+
+    def test_old_style_boolean_serializes_as_bool(self):
+
+        with patch(PATCH_METHOD) as req:
+            req.return_value = BOOLEAN_CONVERSION_MODEL_TABLE_DATA
```

Comment:
This isn't testing the old style, right? `BOOLEAN_CONVERSION_MODEL_TABLE_DATA` has `is_human` defined as a `BOOL`